### PR TITLE
Use correct formatting for commands

### PR DESCRIPTION
**Pull request by pojntfx**
This adds correct Markdown formatting for the commands being used; it currently renders as headings or normal text.

### CODE REVIEW

**Comment by Eonfge** [comment]
@smithmcgriff This would add some reading accessibility

File: README.md [hunk 1]
```diff
@@ -27,23 +36,32 @@ ________________________________________________________________________________
...
+```bash
+sudo passwd root
+```
 
 Then change it.
 
 After that, you'll need to resize the root partition to fill the entirety of your disk. So type:
  
+```bash
 # sudo cfdisk /dev/mmcblkX
```
Comment (thinkofher) [inline README.md]: i think you forgot to remove "#" from that line

Comment (pojntfx) [reply README.md]: Thanks for the review, fixed in faed8e3993d6e0300eebfe6a2819a62127c0f541!

**Comment by FreedomBen** [comment]
I wish I'd looked at this PR earlier.  I also submitted one #4 but I'm thinking the owner is no longer maintaining this or doesn't care about it since your PR has been open for months.  Does anybody want to maintain a fork?  I may maintain mine if not but I have plenty of projects already and don't badly need another :-)  :  https://github.com/FreedomBen/Fedora-on-pinebookpro

I'll undoubtedly add scripts and docs, but I don't plan on wasting time submitting them to an upstream that doesn't want them, so I'll just put them in my fork for now

**Pull request closed without merging**